src/functionaltests/resources/rules/environmentDisabledResponse.json

```diff
@@ -0,0 +1,3 @@
+{
+    "message": "Test environment is not enabled. Please use the test environment for this execution"
```

Comment:
These Error messages will probably be changed in a coming story. Can't we just check the HTTP Error code for the moment?

Reply:
ok, I will not check any error messages